tests/test_backend.py

```diff
@@ -197,20 +197,28 @@ def call_remote_compilation(self):
         torch._dynamo.reset()
 
     @patch("os.path.isfile", new=lambda x: True)
-    @patch("centml.compiler.backend.load_compiled_graph")
-    def test_cgraph_saved(self, mock_load):
+    @patch("builtins.open")
+    @patch("centml.compiler.backend.pickle.load")
+    def test_graph_module_saved(self, mock_load, mock_open):
         mock_load.return_value = MagicMock()
 
-        self.call_remote_compilation()
+        # Don't calculate the model_id since that saves a temp file. Just return "1234"
+        with patch("centml.compiler.backend.Runner._get_model_id", new=lambda x, y: "1234"):
+            self.call_remote_compilation()
```

Comment:
why not use the @patch decorator for this?

Reply:
you're right I should use the decorator